### PR TITLE
Linter: add message about exceptions

### PR DESCRIPTION
**Pull request by bershanskiy**
#### Summary

Add a nice linter message.

#### Test results and supporting details

This is entirely a cosmetic change, it just adds a message to the test output after `All data passed linting!`:
```
Loading and checking files...
Testing browser data...
Testing feature data...
Testing all features together...
All data passed linting!
Linters have some exceptions, please help us remove them!
 Obsolete has 10 exceptions.
  - css.types.length.lh
  - css.types.length.rlh
  - http.headers.Cache-Control.stale-if-error
  - http.headers.Feature-Policy.layout-animations
  - http.headers.Feature-Policy.legacy-image-formats
  - http.headers.Feature-Policy.oversized-images
  - http.headers.Feature-Policy.unoptimized-images
  - http.headers.Feature-Policy.unsized-media
  - svg.attributes.presentation.color-rendering
  - svg.elements.view.zoomAndPan
```
This message increases visibility of exceptions and I aspire to literally guilt-trip myself into fixing them. 😄 

#### Related issues

#16579

### CODE REVIEW

File: test/lint.ts [hunk 1]
```diff
@@ -199,6 +199,23 @@ const main = async (
...
+      console.log(
+        chalk`{yellow Linters have some exceptions, please help us remove them!}`,
+      );
+      for (const linter of linters.linters)
+        if (linter.exceptions) {
+          console.log(
+            chalk`{yellow  ${linter.name} has ${
+              linter.exceptions.length === 1
+                ? '1 excepton:'
+                : `${linter.exceptions.length} exceptions:`
```
Comment (queengooborg) [inline test/lint.ts:212]: We can use the `pluralize()` function here:

```suggestion
            chalk`{yellow  ${linter.name} has ${pluralize('exception', linter.exceptions.length)}`
```

Comment (bershanskiy) [reply test/lint.ts]: Thanks, did not know about this!

Comment (bershanskiy) [reply test/lint.ts]: Ah, I missed a bracket, will fix it shortly...

**Review by queengooborg — APPROVED** [review]
Looks good, thanks!

**Pull request merged**